curriculum/challenges/english/25-front-end-development/lecture-working-with-arrays/67329f508a6ec45b046700b3.md

```diff
@@ -10,6 +10,41 @@ dashedName: what-are-the-key-characteristics-of-javascript-arrays
 
 Watch the video lecture and answer the questions below.
 
+# --transcript--
```

Comment:
we should use double quotes for code examples 

https://contribute.freecodecamp.org/how-to-work-on-coding-challenges/#formatting-seed-code